src/components/UserProfileCard.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="profile_card" @click="$router.push('/login')" v-if="$store.state.currentUser">
+    <img :src="profilePicture" />
+    <div class="detail">
+      <div>{{$store.state.currentUser.displayName}}</div>
+    </div>
+  </div>
+</template>
+
+<script>
+import md5 from 'js-md5'
+export default {
+    name:"UserProfileCard",
+    computed: {
+        profilePicture(){
+            let hash = md5(this.$store.state.currentUser.email);
```

Comment:
Does this use the `LocalStorage` API from browsers? Because AFAIK Apple decided to change their iOS Safari (and thereby transitively affecting other iOS browsers) to limit the `LocalStorage` API third-party data storage expiration day to 7 days.

Reply:
Luckily `localstorage` in this case are only there for test proposes, will remove later so shouldn't be a big deal.

Reply:
Oh and this is not `localstorage`, this is `vuex` which is a vue runtime state management system